packages/cli/src/commands/pg/backups/schedule.ts

```diff
@@ -0,0 +1,112 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import pgHost from '../../../lib/pg/host'
+import {getAttachment} from '../../../lib/pg/fetcher'
+
+type Timezone = {
+  [key: string]: string;
```

Comment:
This can be removed. (see suggestion for line 69)

Reply:
nice, thanks